cloudbuild.yaml

```diff
@@ -19,7 +19,7 @@ steps:
   entrypoint: 'bash'
   args: 
       - '-c'
-      - 'gcloud container clusters get-credentials kubeflow-ci-fairing --zone us-central1-a && pytest -n 8 -v --durations=10 --cov=fairing tests/'
+      - 'gcloud container clusters get-credentials kubeflow-ci-fairing --zone us-central1-a && pytest -n 8 -v --durations=10 --cov=fairing --ignore=tests/integration/azure/ tests/'
 
```

Comment:
Is there going to a way to test the azure integrations in future?

Reply:
We have an experimental CI setup here https://dev.azure.com/kubeflow/kubeflow that runs them. You can see an example of a success run here https://dev.azure.com/kubeflow/kubeflow/_build/results?buildId=38.

From my perspective, it would be great if these could be run in the official CI, but that would require setting up the right secrets in that environment and agreeing how to deal with the small cost associated with actually running things in Azure (cc: @aronchick).